tensorflow_addons/optimizers/adabelief.py

```diff
@@ -113,7 +113,7 @@ def __init__(
             rectify: boolean. Whether to apply learning rate rectification as
                 from RAdam.
             total_steps: An integer. Total number of training steps. Enable
-                warmup by setting a positive value.
+                warmup by setting a value greater than zero.
```

Comment:
I think that it was correct:
http://oeis.org/A000027
http://oeis.org/A001477

Reply:
nice. However, it requires less knowledge to just say "greater than zero". I did not know that discussion

Reply:
There could be some confusion with [signed zero](https://en.wikipedia.org/wiki/Signed_zero) but on integer I think it is hard to be confused. I will merge as it is just an alternative.

Reply:
thanks